addons/ai/functions/fn_waveInit.sqf

```diff
@@ -48,13 +48,15 @@ if(!(_logic getVariable [QGVAR(init),false])) then {
     {
         private _grp = _x;
         private _units = (units _grp) apply {
+            private _unit = _x;
             private _data = [
-                typeOf _x,
-                getPosATL _x,
-                getDir _x,
-                getUnitLoadout _x,
+                typeOf _unit,
+                getPosATL _unit,
+                getDir _unit,
+                getUnitLoadout _unit,
                 -1,
-                []
+                [],
+                ALL_AI_FEATURES apply {[_x,_unit checkAIFeature _x]}
```

Comment:
Replace with `ALL_AI_FEATURES select {!(_unit checkAIFeature _x)}`
Only store what we need, aka all disabled features